openshift-knative-operator/pkg/serving/extension.go

```diff
@@ -132,7 +134,13 @@ func (e *extension) Reconcile(ctx context.Context, comp v1alpha1.KComponent) err
 	return nil
 }
 
-func (e *extension) Finalize(context.Context, v1alpha1.KComponent) error {
+func (e *extension) Finalize(ctx context.Context, ks v1alpha1.KComponent) error {
+	// Delete the ingress namespaces (manifestival doesn't delete namespaces)
```

Comment:
Dug into a rabbit hole and figured out what was wrong. Tl;dr: We need the manual delete anyway because there is a bug in manifestival. See the new comment.